src/components/settings/SettingsLMSTab/tests/ExistingLMSCardDeck.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  render, screen,
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import ExistingLMSCardDeck from '../ExistingLMSCardDeck';
+
+const mockOnClick = jest.fn();
+const configData = [
+  {
+    channelCode: 'BLACKBOARD',
+    id: 1,
+    isValid: {},
+    active: true,
+    displayName: 'foobar',
+  },
+];
+
+const incompleteConfigData = [
+  {
+    channelCode: 'BLACKBOARD',
+    id: 2,
+    isValid: { missing: ['client_id'] },
+    active: false,
+    displayName: 'barfoo',
+  },
+];
+
+describe('<ExistingLMSCardDeck />', () => {
+  it('renders active config card', () => {
+    render(
+      <ExistingLMSCardDeck
+        configData={configData}
+        editExistingConfig={mockOnClick}
+      />,
+    );
+    expect(screen.getByText('Active')).toBeTruthy();
```

Comment:
I think toBeTruthy() may be fine for what you need to test here, but I would consider using `toBeInTheDocument()` which comes from jest matchers https://github.com/testing-library/jest-dom#tobeinthedocument . I believe this one actually checks for its presence in the dom vs toBeTruthy just checks if the element is valid (but it does not have to have been added to the dom etc)